pkg/environment/api/project.go

```diff
@@ -272,9 +264,8 @@ func validateCreateProjectRequest(req *environmentproto.CreateProjectRequest, lo
 		}
 		return dt.Err()
 	}
-	// TODO Once we support new create project API requiring name instead of id, we should validate name only.
 	name := strings.TrimSpace(req.Command.Name)
-	if !projectNameRegex.MatchString(name) && !projectIDRegex.MatchString(req.Command.Id) {
+	if !projectNameRegex.MatchString(name) {
```

Comment:
Please remove the regex for the name. Otherwise, we can't use space, etc.

Reply:
For the name field, please use this as a reference.

https://github.com/bucketeer-io/bucketeer/blob/fcacff8f1a28cca27f810ec16a3e670700d15c55/ui/web-v2/apps/admin/src/pages/feature/formSchema.ts#L33

Reply:
@cre8ivejp 
Thank you, I fixed it 👍